packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,340 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
-      </va-tree-view>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
+    </VbCard>
+    <VbCard title="Stateless">
+      <va-tree-view :nodes="nodes" :stateful="false" />
```

Comment:
This demo doesn't seem to be working.

![image](https://user-images.githubusercontent.com/5394573/191755410-3ee707ce-6d35-46cb-9665-9bc1c746e696.png)